.eslintrc.json

```diff
@@ -14,7 +14,8 @@
   "globals": {
     "flowchart": true, // Blueprints support lib
     "__": true, // Zanata parsing cue
-    "N_": true // Zanata parsing cue
+    "N_": true, // Zanata parsing cue
+    "$window": true
```

Comment:
let me remove that commit -- 0911e7c134d4efe823f44dbab17b133c7444d30e so we know the exact CC error